### PR TITLE
[EOS-20036] run ha component in sequence primary and then secondaries

### PR DESCRIPTION
**Pull request by mazinamdar**
Signed-off-by: mazinamdar <mazhar.inamdar@seagate.com>

### CODE REVIEW

**Review by sradhanand-pati — APPROVED** [review]

File: lr-cli/cortx_setup/commands/cluster/config/component.py [hunk 1]
```diff
@@ -67,6 +76,8 @@ def run(self, component_group):
             # Getting provisioner states for deployment
             # noncortx_components = get_provisioner_states()
             cortx_components = get_cortx_states()
+            primary = local_minion_id()
+            secondaries = f"not {primary}"
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/cluster/config/component.py:80]: How does `secondaries` incorporate only `minion_ids` except for primary?

File: lr-cli/cortx_setup/commands/cluster/config/component.py [hunk 2]
```diff
@@ -79,13 +90,23 @@ def run(self, component_group):
...
                         stages=['config.config', 'config.init_mod']
                     )
-                self.logger.debug(f"Deploying ha components on nodes")
+                self.logger.debug(f"Deploying cortx ha components on {primary}")
                 self._configure(
                     ['ha.cortx-ha'],
                     stages=[
-                        'config.post_install','config.prepare',
-                        'config.config', 'config.init_mod']
+                        'config.post_install', 'config.prepare',
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/cluster/config/component.py:97]: **Below phases should be executed in parallel on all nodes:**
`'config.post_install', 'config.prepare',`

File: lr-cli/cortx_setup/commands/cluster/config/component.py [hunk 3]
```diff
@@ -79,13 +90,23 @@ def run(self, component_group):
...
+                        'config.config', 'config.init_mod'],
+                    targets=primary
                 )
+                self.logger.debug(f"Deploying cortx ha components on {secondaries}")
+                self._configure(
+                    ['ha.cortx-ha'],
+                    stages=[
+                        'config.post_install', 'config.prepare',
+                        'config.config', 'config.init_mod'],
+                    targets=secondaries
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/cluster/config/component.py:107]: **Below phases should be executed in sequence on primary-secondary nodes:**
`'config.config', 'config.init_mod'`

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Pull request merged**